app/controllers/course/assessment/submission_question/comments_controller.rb

```diff
@@ -52,6 +57,10 @@ def last_post_from(submission_question)
     submission_question.posts.ordered_topologically.flatten.select(&:id).last
   end
 
+  def discussion_topic
+    @submission_question.discussion_topic
```

Comment:
Did you run into any trouble with `delegate`?

Reply:
Not really. Did this to be more consistent with the rest of the controllers - switched to this to test whether it caused the issue here -> https://github.com/Coursemology/coursemology2/pull/2743/files#diff-efa56180d3c31f9002c14d3d473607d7R24